cafes/views.py

```diff
@@ -61,4 +66,64 @@ def post(self, request, review_id):
             user                 = request.user
     )
 
-        return JsonResponse({'MESSAGE':'SUCCESS'}, status=201)
\ No newline at end of file
+        return JsonResponse({'MESSAGE':'SUCCESS'}, status=201)  
+            
+class CafeInformationView(View):
```

Comment:
`Info`, `Information`같은 용어는 사용하지 않는 습관을 만드시는게 좋습니다~
`CafeView`만으로 의미 전달이 충분하기 때문이죠~

```suggestion
class CafeView(View):
```